spi/transfer-spi/src/test/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/TransferProcessTest.java

```diff
@@ -142,6 +146,26 @@ void verifyProvisioningComplete() {
         assertTrue(process.provisioningComplete());
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = { 0, 100, 200, 300, 400, 500, 600, 700, 850, 900, 1000, 1200 })
```

Comment:
Same as before, the `@EnumSource` could do a good job here, in this case the `mode = EXPLICIT` could be used to make it more succint